reports/utils.py

```diff
@@ -37,3 +37,11 @@ def midnight_validator(inputstr):
 def generate_random_filename(suffix='.xlsx'):
     return ''.join(
         random.choice(string.ascii_lowercase) for i in range(12)) + suffix
+
+
+def sizeof_format(num, suffix='B'):
+    for unit in ['', 'Ki', 'Mi', 'Gi', 'Ti', 'Pi', 'Ei', 'Zi']:
+        if abs(num) < 1024.0:
+            return "%3.1f%s%s" % (num, unit, suffix)
+        num /= 1024.0
+    return "%.1f%s%s" % (num, 'Yi', suffix)
```

Comment:
We've got a function in Django to do this for us: https://docs.djangoproject.com/en/dev/ref/templates/builtins/#filesizeformat